.github/workflows/sycl_precommit.yml

```diff
@@ -4,6 +4,13 @@ on:
   pull_request:
     branches:
     - sycl
+    # Do not run builds if changes are only in the following locations
+    paths-ignore:
+    - 'devops/containers/**'
+    - 'devops/scripts/install_drivers.sh'
+    - 'devops/scripts/install_build_tools.sh'
+    - 'sycl/doc/**'
+    - 'clang/docs/**'
```

Comment:
```suggestion
    - 'sycl/gdb/**'
    - 'sycl/examples/**'
    - 'clang/docs/**'
    - 'buildbot/**'
    - '*.md' # not sure if it's correct syntax, but it aims to ignore changes in README.md, CONTRIBUTING.md, etc. Ideally we should ignore changes in any Markdown, ReST, .txt files in any sub-directory.
```

Reply:
What about buildbot files and sycl/examples?

Reply:
Files in both directories can affect builds, so, I'd prefer them to go through full build cycle.

Reply:
Which buildbot files affect GitHub actions builds?

Reply:
configure.py

Reply:
I think we should move scripts shared between different CI systems to a separate directory.